packages/ui/src/components/address.tsx

```diff
@@ -46,18 +46,16 @@ Address.Country = function Country(props: SelectProps) {
   return (
     <Select
       defaultValue={country}
-      onChange={(e) => {
-        setCountry(e.currentTarget.value as SupportedCountry);
+      onChange={(event) => {
+        setCountry(event.currentTarget.value as SupportedCountry);
       }}
       {...props}
     >
-      {COUNTRIES.map((country) => {
-        return (
-          <option key={country.abbreviation} value={country.abbreviation}>
-            {country.name}
-          </option>
-        );
-      })}
```

Comment:
Let's keep the same syntax style (ie: using curly braces w/ return statement instead of the shorthand), apply this everywhere please!